client/utils/automation.js

```diff
@@ -0,0 +1,21 @@
+export const isSupportedByResponseCollector = ({ at, browser }) =>
+    at?.name === 'NVDA' && browser?.name === 'Chrome';
+
+export const isBot = user => user?.username?.toLowerCase().slice(-3) === 'bot';
+
+export const isAssertionValidated = assertion => {
+    return (
+        assertion.passed ||
+        (assertion.failedReason &&
+            assertion.failedReason !== 'AUTOMATED_OUTPUT')
+    );
+};
+
+// TODO: Stub, support for more bot users should be added
+export const getBotUserFromAtBrowser = (at, browser) => {
```

Comment:
Since the object returned by this function doesn't fully conform to the "user" type, and since the only call site just needs the name, I think it'd be more clear to define `getBotUsernameFromAtBrowser`, instead.